esrally/racecontrol.py

```diff
@@ -107,19 +109,26 @@ def receiveUnrecognizedMessage(self, msg, sender):
     def receiveMsg_Setup(self, msg, sender):
         self.start_sender = sender
         self.cfg = msg.cfg
+        assert self.cfg is not None
         self.coordinator = BenchmarkCoordinator(msg.cfg)
         self.coordinator.setup(sources=msg.sources)
         self.logger.info("Asking mechanic to start the engine.")
         self.mechanic = self.createActor(mechanic.MechanicActor, targetActorRequirements={"coordinator": True})
         self.send(
             self.mechanic,
             mechanic.StartEngine(
-                self.cfg, self.coordinator.metrics_store.open_context, msg.sources, msg.distribution, msg.external, msg.docker
```

Comment:
Don't know why this line has been left until now, but black 23.3.0 reformats it now.